packages/core/include/soss/Message.hpp

```diff
@@ -18,104 +18,8 @@
 #ifndef SOSS__MESSAGE_HPP
 #define SOSS__MESSAGE_HPP
 
-#include <functional>
-#include <map>
-#include <memory>
-#include <string>
+#include <dds/core/xtypes/xtypes.hpp>
```

Comment:
The purpose of `dds/core/xtypes/xtypes.hpp` is to bring all the necessary of xtypes to the soss ecosystem.

I do not know if you are referring to make separated *includes*, (one for each type for example).  If this is the case, the aim is to simplify the user management: if he writes the `xtypes` namespace, he has access to the public interface of the module. We have chosen this strategy because all xtypes entities are very coupled.

Reply:
I was referring to Message.hpp, as it currently contains an include... maybe just change all includes that point to Message.hpp to xtypes.hpp and remove Message.hpp?

Reply:
I left the `Message.hpp` file there in order to pack some others inclusions that the messaging might need, or maybe some future utility over the `xtypes` that `soss-core` could offer. The user only needs to carry about the `Message.hpp` inclusion.

Reply:
ok, we can keep it like that then.